authentification/domain/authentification.go

```diff
@@ -38,7 +37,7 @@ type Auth struct {
 func HashPassword(password string) (string, error) {
 	bytes, err := bcrypt.GenerateFromPassword([]byte(password), 14)
 	if err != nil {
-		log.Fatal(err)
+		return "", fmt.Errorf("Password error: %v", err.Error())
```

Comment:
pas besoin de .Error(), %v le fait tout seul

ou alors:
```suggestion
		return "", fmt.Errorf("Password error: %s", err.Error())
```